docs/content.zh/docs/ops/metrics.md

```diff
@@ -1428,6 +1428,16 @@ Note that the metrics are only available via reporters.
       <td>Maximum recorded duration of a single consecutive period of the task being in the hard back pressure state in the last sampling period. Please check softBackPressuredTimeMsPerSecond and hardBackPressuredTimeMsPerSecond for more information.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td>mailboxThroughputPerSecond</td>
+      <td>The number of actions processed from the task's mailbox per second over all actions which includes, e.g., checkpointing, timer, or cancellation actions.</td>
+      <td>Meter</td>
+    </tr>
+    <tr>
+      <td>mailboxThroughputLatencyUs</td>
+      <td>The latency in processing actions from the task's mailbox, i.e., an action's waiting time in the mailbox, in microseconds for actions with standard (default) priority, e.g., checkpointing actions.</td>
```

Comment:
I would mention that it is being sampled periodically, so it might not be fully accurate.